jest.setup.js

```diff
@@ -0,0 +1 @@
+jest.setTimeout(20000)
```

Comment:
When running the tests locally this file (`jest.setup.js`) isn't copied over into the docker container, I'm not sure how it is working in Jenkins?

The file can be copied over by adding `- ./jest.setup.js:/home/node/jest.setup.js` to `docker-compose.test.yaml`. However, another option is to increase the test timeout for the specific test causing the problem. It seems that adding the timeout to `10000` (10 seconds) to the `beforeEach` on `message-receiver.test.js.` fixes the timeout problem. In order to avoid increasing the timeout for all tests and having to make the additional changes to copy the file into the container I suggest this is the easier option. It also has the benefit of making it obvious which tests need the additional time.

What do you think?

Reply:
In other circumstances I would agree with this suggestion.  But as we're using Jest for contract and integration tests, with heavy container usage and the latency from Azure Service Bus, then the default of five seconds is almost always breached by a significant number of tests.

I think the Jest file is the better approach as it avoids the need to micromanage timeouts in each test suite.  Especially as it could involve some trial and error when it comes to execution time locally vs pipeline and you could see some unexpected failures.

In the `ffc-demo-payment-service` we did quite a bit of work trying to understand what setup was the most appropriate.  So for test structure I'd like us to mirror that where possible.